lib/miq_automation_engine/engine/miq_ae_event.rb

```diff
@@ -121,6 +121,7 @@ def self.call_automate(obj, attrs, instance_name, options = {})
     raise "A user is needed to raise #{instance_name} to automate. [#{obj.class.name}] id:[#{obj.id}]" unless user
 
     q_options = {
+      :zone         => Zone.determine_queue_zone(options),
```

Comment:
@tinaafitz This is already called during `MiqQueue.put`?  Is this really needed?